spec/spec_helper.rb

```diff
@@ -3,6 +3,9 @@
 require_relative "support/simplecov"
 require "sidekiq"
 
+# Silence ruby warnings
+$TESTING = true # rubocop:disable Style/GlobalVars
```

Comment:
Hmm, I actually think it's helpful to see them, especially if we need to deal with deprecation warnings for instance

Reply:
$TESTING is a sidekiq global variable. It does not silence Ruby warnings, when ruby executed with `-w` argument or `RUBYOPT=-w` environment variable.

See Ruby globals: https://docs.ruby-lang.org/en/2.7.0/globals_rdoc.html

Reply:
This is a different kind of warnings:

```
RUBYOPT=-W0 rspec
gems/sidekiq-5.2.10/lib/sidekiq/processor.rb:66: warning: global variable `$TESTING' not initialized
gems/sidekiq-5.2.10/lib/sidekiq/launcher.rb:63: warning: global variable `$TESTING' not initialized
gems/sidekiq-5.2.10/lib/sidekiq/cli.rb:17: warning: global variable `$TESTING' not initialized
```